src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final String DATE_PATTERN = "dd.MM.yyyy";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDay = LocalDate.parse(dateFrom, DateTimeFormatter.ofPattern(DATE_PATTERN));
+        LocalDate finalDay = LocalDate.parse(dateTo, DateTimeFormatter.ofPattern(DATE_PATTERN));
```

Comment:
The `DateTimeFormatter` instance created from `DATE_PATTERN` is used multiple times in the method. It would be more efficient to create a static final `DateTimeFormatter` constant field and use it throughout the class instead of calling `DateTimeFormatter.ofPattern(DATE_PATTERN)` multiple times.